### PR TITLE
Removed "bc-" prefix of barclamps

### PR DESCRIPTION
**Pull request by vwallfahrer**

### CODE REVIEW

File: scripts/barclamp_mgmt_lib.rb [hunk 1]
```diff
@@ -94,6 +94,18 @@ def catalog
...
+      [ File.join(bc_path, '..', name, 'chef', 'data_bags', 'crowbar', "template-#{name}.json"), \
+        File.join(bc_path, '..', "barclamp-#{name}", 'chef', 'data_bags', 'crowbar', "template-#{name}.json"), \
+        File.join(bc_path, '..', '..', 'chef', 'data_bags', 'crowbar', "template-#{name}.json") ].each do |f|
+        next unless File.exist? f
+        s = JSON::load File.open(f, 'r')
+        description = s['description'] unless s.nil?
+        break if description
+      end
+    end
+    # template = File.join bc_path, name,
```
Comment (vuntz) [inline scripts/barclamp_mgmt_lib.rb]: I think you had an outdated version, because this code got dropped.

File: scripts/barclamp_mgmt_lib.rb [hunk 2]
```diff
@@ -94,6 +94,8 @@ def catalog
     description = bc['barclamp']['description']
     puts "Warning: Barclamp #{name} has no description!" if description.nil?
     display = bc['barclamp']['display']
+    end
```
Comment (dirkmueller) [inline scripts/barclamp_mgmt_lib.rb]: syntax error..

File: scripts/barclamp_mgmt_lib.rb [hunk 3]
```diff
@@ -94,6 +94,7 @@ def catalog
     description = bc['barclamp']['description']
     puts "Warning: Barclamp #{name} has no description!" if description.nil?
     display = bc['barclamp']['display']
+    # template = File.join bc_path, name,
```
Comment (tboerger) [inline scripts/barclamp_mgmt_lib.rb]: What is this?

**Comment by vuntz** [comment]
+1

**Comment by toabctl** [comment]
+1

**Pull request merged**